src/dune/__main__.py

```diff
@@ -220,8 +232,11 @@ def load_all_modules_from_dir(plugin_dir):
             elif args.bootstrap_system:
                 dune_sys.bootstrap_system(False)
 
-            elif args.bootstrap_system_full:
-                dune_sys.bootstrap_system(True)
+            elif args.bootstrap_system_full is not None:
+                if len(args.bootstrap_system_full) > 3:
+                    sys.stderr.write("Error: --bootstrap-system-full should have at most 3 arguments\n")
```

Comment:
```
beauty$ ./dune --bootstrap-system-full arg1 arg2 arg3 arg4 arg5 && echo success
Error: --bootstrap-system-full should have at most 3 arguments
success
```
I think our fail path should return an error in case this is scripted. 

Maybe this would work?
```suggest
                    parser.exit_with_help_message("Error: --bootstrap-system-full should have at most 3 arguments\n")
```

Reply:
Thanks for finding it, this is much better

Reply:
Thank, fixed